### PR TITLE
DOCS: Updated FUSE docs

### PR DESCRIPTION
**Pull request by hosh**
Additional instructions on files that need to have permissions changed.

### CODE REVIEW

File: docs/fuse.md [hunk 1]
```diff
@@ -26,6 +28,12 @@ go get github.com/jbenet/go-fuse-version/fuse-version
 
 If you run into any problems installing FUSE or mounting IPFS, hop on IRC and speak with us, or if you figure something new out, please add to this document!
 
+You may also have to add your username to the `operator` group:
```
Comment (jbenet) [inline docs/fuse.md]: could we describe more why this has to happen? or if there is another way to do it? some users will not want to modify user groups.

File: docs/fuse.md [hunk 2]
```diff
@@ -40,7 +48,17 @@ Then change permissions on the fuse config:
 sudo chown <username>:<groupname> /etc/fuse.conf
 ```
 
-(note `<groupname>` should probably be `fuse`)
+You may also have to change `/dev/fuse`:
```
Comment (jbenet) [inline docs/fuse.md]: `change permissions on` *

**Comment by jbenet** [comment]
some comments, but after those, this LGTM!

File: docs/fuse.md (hunk 1 above)
Comment (hosh) [inline docs/fuse.md]: I'm not sure. I had not tested it without it. `/dev/osxfuse*` all have `root:operator`. Might still work without it, which is why I said "might".

Comment (jbenet) [inline docs/fuse.md]: haha fair. It works for me without (am not an operator, am in `staff` though). But i did have to:

```
sudo chown jbenet /ipfs
sudo chown jbenet /ipns
```

**Comment by jbenet** [comment]
ping

**Comment by cryptix** [comment]
Since this is still open, we could add the `nofuse` build tag requirement for windows?

**Comment by whyrusleeping** [comment]
> Since this is still open, we could add the nofuse build tag requirement for windows?
:+1:

**Comment by hosh** [comment]
What's the `nofuse` build tag?

**Comment by jbenet** [comment]
Some files have a header that looks like:
```
// +build linux darwin freebsd
// +build nofuse
```

the `nofuse` build tag is: `// +build nofuse`

**Comment by whyrusleeping** [comment]
@hosh ping! what is the status here?

**Comment by hosh** [comment]
@whyrusleeping I'll get some updates and a rebase in tonight

**Comment by hosh** [comment]
So I am not entirely clear what the build nofuse build requirement for windows is -- specifically, FUSE isn't supported in the docs. Isn't saying that FUSE isn't supported on Windows sufficient?

**Pull request merged**